src/main/java/edu/kis/powp/jobs2d/transformations/changedriver/ApplyTransformationToNewDriver.java

```diff
@@ -20,9 +20,6 @@ public static void applyTransformation(Job2dDriver transformation) {
         DrawerFeature.getDrawerController().clearPanel();
         LinesRecorder.getLinesRecorder().clearLines();
 
-        CommandsFeature.getCommandManager().setCurrentCommand(RecordCommandsForTransformation.getRecordedCommand());
-
-        CommandsFeature.getCommandManager().getCurrentCommand().execute(DriverFeature.getDriverManager().getCurrentDriver());
-
+        RecordCommandsForTransformation.getRecordedCommand().execute(DriverFeature.getDriverManager().getCurrentDriver());
```

Comment:
You have forgotten to clean up imports (import edu.kis.powp.jobs2d.command.manager.ICommandManager).

For a cleaner code, this command should be divided into at least 3 lines.